### PR TITLE
Aggregated stats widgets

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See https://trello.com/c/suWzyusJ for details.

The npm dependancies have been updated as much as I dare (i.e. React only pushed to latest 15.x, and d3 needed to stay in the 3.x realm).  The dev dependancies are largely untouched, due to it being a significant effort to update webpack and the test runners, which didn't seem worth it for a repo we'll be deprecating.

With the agg stats being generated in `viz` now, I've removed all (I think) of the stat related code from the source and tests here.  As well, the sidebar in the basics view has been removed, and is now rendered in `blip`

One other small change to note is that we're now rendering all bg data using the 5-way colors so that they match the new stats (i.e. `veryLow` and `veryHigh` are no longer rendered in the same colors as `low` and `high`)

Related PRs
---
tidepool-org/blip#513
tidepool-org/viz#120

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
[Standardize Aggregated Agg Stats Widgets [data viz] [general]](https://trello.com/c/suWzyusJ/441-standardize-aggregated-agg-stats-widgets-data-viz-general)

File: plugins/blip/basics/components/chart/WrapCount.js [hunk 1]
```diff
@@ -39,7 +38,11 @@ var WrapCount = React.createClass({
     );
   },
   generateDots: function(start, end, dotSize, pad) {
-    pad = pad || 1.5;
+    pad = pad || 0;
```
Comment (krystophv) [inline plugins/blip/basics/components/chart/WrapCount.js]: These edits seem like they may not be entirely settled, between the commented code and the immediate reassignment of `pad`.

Comment (cbwebdevelopment) [reply plugins/blip/basics/components/chart/WrapCount.js]: Yeah, thanks.  I'll clean this up.

**Comment by cbwebdevelopment** [comment]
Thanks, Chris.  :)

I've removed the unused padding calculations.

**Pull request merged**